pages/_document.js

```diff
@@ -20,6 +20,18 @@ export default function Document() {
         user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/ */}
         <link rel="manifest" href="/manifest.json" />
 
+        {/* Google tag (gtag.js) */}
```

Comment:
Does this add GA to the whole site, every page?

Reply:
Yes this adds it to every page.